src/blob_storage/postgres/schema.clj

```diff
@@ -25,7 +110,8 @@
                 (create-table sqdb :blobs
                               (if-not-exists true)
                               (column :id :varchar :length 150 :primary-key? true)
-                              (column :blob :bytea :not-null? true)
```

Comment:
La columna `blob` ya no puede ser `NOT NULL`. Ahora veo que me falta agregar un check constraint como el siguiente:

```sql
ALTER TABLE blobs ADD CONSTRAINT blob_or_oid_set CHECK (blob IS NOT NULL or oid IS NOT NULL)
```

Reply:
Para el caso de prisma ya hay un fixer que agrega la columna y acomoda los constraints.